cryptus-app/components/wallet_viewer/collection_details/collection_viewer/nft_block/nft_block.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect, useState } from "react";
+import s from "./nft_block.module.scss";
+
+export default function NftBlock(props: { name: string; nft: string }) {
+  return (
+    <div className={s.container}>
+      <div className={s.none}>
```

Comment:
put this div inside the NftPictureBlockInCollection function instead, so the main code is cleaner